scripts/ci/test_plan.py

```diff
@@ -292,7 +290,7 @@ def parse_args():
     parser.add_argument('-m', '--modified-files', default=None,
             help="File with information about changed/deleted/added files.")
     parser.add_argument('-o', '--output-file', default="testplan.csv",
```

Comment:
testplan.json

Reply:
keeping this as before to work around github actions and the type of workflow we use, otherwise the new twister wont be used and tested... this will be changed once this PR is ready to be merged or in a following fixup PR